internal/compiler/generator/rust.rs

```diff
@@ -2424,6 +2424,20 @@ fn compile_builtin_function_call(
             let window_adapter_tokens = access_window_adapter_field(ctx);
             quote!(slint::private_unstable_api::re_exports::WindowInner::from_pub(#window_adapter_tokens.window()).set_text_input_focused(#(#a)*))
         }
+        BuiltinFunction::MapPointToWindow => {
+            if let [Expression::PropertyReference(pr), Expression::Struct { values, .. }] =
+                arguments
+            {
+                let item_rc = access_item_rc(pr, ctx);
+                let x_expr = compile_expression(values.get("x").unwrap(), ctx);
+                let y_expr = compile_expression(values.get("y").unwrap(), ctx);
+                quote!(
+                    (*#item_rc).map_to_window(slint::private_unstable_api::re_exports::LogicalPoint::new(#x_expr as _, #y_expr as _)).to_untyped()
```

Comment:
Why do we have to pass x_expr and y_expr  there?
Shouldn't we pass (0,0) ?

Reply:
Ah yes, I can remove these arguments and just do the addition of the item local coordinates in the generated code.